src/components/shared/forms/fileInput.js

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Btn } from 'components/shared';
+import { joinClasses, svgs } from 'utilities';
+
+import './styles/select.css';
+import './styles/fileInput.css';
+
+export const FileInput = ({ label, acceptTypes, className, ...props }) => {
+  let fileInput = null;
```

Comment:
Per react docs:
Legacy API: String Refs
If you worked with React before, you might be familiar with an older API where the ref attribute is a string, like "textInput", and the DOM node is accessed as this.refs.textInput. We advise against it because string refs have some issues, are considered legacy, and are likely to be removed in one of the future releases.
I changed it to use their current recommended method